plugin/mutate/doc/design/options.md

```diff
@@ -0,0 +1,220 @@
+# Options and flags
+partof: DOC-plugin_mutate
+
+This document aims to improve usability for the *Mutate* plugin in Dextool by explaining flags and options in more detail. The following sections are divided into the current existing commands for *Mutate*.
+
+The flags for the different commands will be listed in the following format:
+
+- *--flag-long-version [-flag-short-version] : Description*
+    - *input_example*
+
+**Note:** Several of the commands listed below can be set in the configuration-file as well. It is recommended to generate an *.toml*-file and set the corresponding fields in that file instead of having long chains of commands to execute in the terminal (it is usually more simple that way, reduces clutter in the terminal and increase usability for coming test sessions). The information listed below can also be viewed by executing *dextool mutate --help* in a terminal window, but with a shorter and less detailed explanation.
+
+## General
+*This options are general and can be used for all the commands in Mutate.*
+
+
+- --config [-c] : Load configuration (default: .dextool_mutate.toml). Can be used if another project contains a configuration-file which the users wants to reuse.
+
+
+- --db : Specify which sqlite3 database to use for the mutation testing (default: dextool_mutate.sqlite3). This option could be used when several databases exists for i.e. different versions of the code, or simply because the user have chosen to create several databases for result.
+
+
+- --out : Path used as the root for mutation/reporting of files (default: .).
+
+
+- --help [-h] : Display the help information in terminal window (less detailed). By writing, for example, *dextool mutate analyze --help*, the user will display the help for the analyze command.
+
+## Admin
+*Admin-mode for the plugin. Is used by to execute administrative commands and to initialize/setup mutation testing for a project.*
+
+
+- --dump-config : Dump the detailed configuration in the terminal. Could be used to create and setup your own configuration-file.
+
+
+- --init : Create an initial configuration to use in the current workspace. This command is used when the mutation testing is setup for a specific project the first time.
+
+
+- --mutant : Mutants to operate on.
+    - *any* : Any mutant generated.
+    - *ror* : Relational Operator Replacement.
+    - *rorp* : Relational Operator Replacement (Pointer).
+    - *lcr* : Logical Connector Replacement.
+    - *lcrb* : Logical Connector Replacement (Bit-wise).
+    - *aor* : Arithmetical Operator Replacement.
+    - *uoi* : Unary Operator Insertion.
+    - *abs* : Absolute Value Insertion.
+    - *sdl* : Statement Deletion.
+    - *cor* : Conditional Operator Replacement.
+    - *dcc* : Decision/Condition Coverage.
+    - *dcr* : Decision/Condition Replacement.
```

Comment:
replacement -> requirement
The operator *requires* that you kill the mutants that it generates for case-branches compared to dcc where you only need to visit them for the *bomb* to go off.